root/app/updater.py

```diff
@@ -181,38 +191,72 @@ def get_image(repo):
         category=categories,
         stable=stable,
         deprecated=deprecated,
+        stars=repo.stargazers_count,
+        monthly_pulls=scarf_data.get(project_name, None),
         tags=tags,
         architectures=get_architectures(readme_vars),
-        changelog=get_changelogs(readme_vars),
+        changelog=changelog,
         config=config,
     )
 
 def update_images():
     with KeyValueStore(invalidate_hours=INVALIDATE_HOURS, readonly=False) as kv:
         is_current_schema = kv.is_current_schema("images", IMAGES_SCHEMA_VERSION)
         if ("images" in kv and is_current_schema) or CI == "1":
-            print(f"{datetime.datetime.now()} - skipped - already updated")
+            print(f"{datetime.datetime.now()} - images skipped - already updated")
             return
         print(f"{datetime.datetime.now()} - updating images")
         images = []
+        scarf_data = json.loads(kv["scarf"])
         repos = gh.get_repos()
         for repo in sorted(repos, key=lambda repo: repo.name):
-            image = get_image(repo)
+            image = get_image(repo, scarf_data)
             if not image:
                 continue
             images.append(image)
         
         data = ImagesData(repositories=Repository(linuxserver=images))
-        last_updated = datetime.datetime.now(datetime.timezone.utc).isoformat(' ', 'seconds')
+        last_updated = datetime.datetime.now(datetime.timezone.utc).isoformat(" ", "seconds")
         response = ImagesResponse(status="OK", last_updated=last_updated, data=data)
         new_state = response.model_dump_json(exclude_none=True)
         kv.set_value("images", new_state, IMAGES_SCHEMA_VERSION)
         print(f"{datetime.datetime.now()} - updated images")
 
+def get_monthly_pulls():
```

Comment:
Is this getting total downloads or unique sources?

Reply:
Based on the numbers it seems like the total downloads, not unique.